### DISCUSSION TITLE
jobExecutor

### DISCUSSION THREAD

**Discussion by formikaro** [inlife/nexrender#875] [Q&A]
Hello,

great project, really enjoy building things with it. I noticed that beginning Febuary this year the 'jobExecutor' field is not populated anymore. Is this due to a change on the nexrender server or is this more a config issue on my end? (render machines are running on windows 11, AE 23.2.1)
Any hint appreciated, 
Cheers
Johannes

**Comment by inlife** [comment]
Hey @formikaro 

Could you help me identify 2 things, what did this field do, and which version of nexrender did it work correctly in?

Seems like a regression could have happened, but I'm not sure where to look.

**Reply by formikaro** [reply]
Hey @inlife,

thanks for your response. 

There were two fields actually **executor** and **jobExecutor** and they contained the ip of the machine the nexrender worker was running. Unfortunately I cannot track back exactly which version was used then but as this was in February this year it has to be a recent one.

```
},
"creator": "::ffff:127.0.0.1",
"updatedAt": "2023-02-01T09:53:42.408Z",
"createdAt": "2023-02-01T09:51:11.183Z",
"**executor**": "::ffff:10.0.0.13",
"startedAt": "2023-02-01T09:53:14.287Z",
"renderProgress": 91,
"error": null,
"finishedAt": "2023-02-01T09:53:42.234Z",
"errorAt": null,
"jobCreator": "::ffff:127.0.0.1",
"**jobExecutor**": "::ffff:10.0.0.13"
},
```

**Reply by inlife** [reply]
Thanks!

From what I see, they are still there, and everything should be working correctly. https://github.com/inlife/nexrender/blob/master/packages/nexrender-server/src/routes/jobs-pickup.js#L52

Also, I do not think there are configuration files related to this.

**Reply by formikaro** [reply]
Do you have any pointers for me why this could fail then so I know in which direction to dig? Shouldn't the keys still be part of the Json, if only empty?
